DSE_6.8_Release_Notes.md

```diff
@@ -4,6 +4,45 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
 
 Release notes of versions prior to 6.8.4 can be found [here](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/releaseNotes/RNdse.html).
 
+# Release notes for 6.8.49
+10 June 2024
+
+## Components versions for DSE 6.8.49
+* Apache Solr™ 6.0.1.4.2964
+* Apache Spark™ 2.4.0.30
+* Apache TinkerPop™ 3.4.14-20240307-bcc67d14
+* Apache Tomcat® 8.5.94
+* DSE Java Driver 1.10.0-dse-20240520&ast; (DSE *internal-only* version)
+* Netty 4.1.100.1.dse
+* Spark JobServer 0.8.0.54
+
+**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
+If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.
+
+## 6.8.49 DSE Core
+* Prevents a Java driver request timeout for `drain()` operations executed via Management API by setting the request timeout to 0. The request timeout change only affects the internal Java driver used in Management API and is only set to 0 explicitly for the `drain()` operation. (DSP-23994)
+
+## 6.8.49 DSE Cassandra
+* Secondary Index: Don't fail queries if one node is not available. (DSP-24163)
```

Comment:
I would propose to phrase it a bit differently as currently it's difficult to guess what it does from the user perspective: how about _Introduced a flag that allows skipping querying nodes that do not have ready secondary indexes". This behaviour can be enabled by setting `-Ddse.cassandra.index.check_index_status=true`_? @roxananeo as the ticket author, what do you think?